docs/operator/api.md

```diff
@@ -1602,7 +1602,9 @@ ProxyAuth represent proxy auth config Only VictoriaMetrics scrapers supports it.
 
 ## RelabelConfig
 
-RelabelConfig allows dynamic rewriting of the label set, being applied to samples before ingestion. It defines `<metric_relabel_configs>`-section of configuration. More info: https://prometheus.io/docs/prometheus/latest/configuration/configuration/#metric_relabel_configs
+RelabelConfig allows dynamic rewriting of the label set; it can be applied:
```

Comment:
`RelabelConfig` is the same structure referenced by  both `relabel_config` and `metric_relabel_configs` so expanding on both options here; alternatively we could be more generic here and just say "RelabelConfig allows dynamic rewriting of the label set".